tools/bigquery-zos-mainframe-connector/USING_WITH_GCS.md

```diff
@@ -0,0 +1,89 @@
+# Using with Cloud Storage
+
```

Comment:
link to the readme for the gRPC service along with a minimal introduction.

Reply:
I think it makes sense to put a description of how teams may deploy their own gRPC servers if they want dedicated transcoding capacity.

Reply:
also helpful here is the number of parallel requests served by the gRPC service of a typical size. (maybe 8 cores, 16GB) and what happens when there are excess requests due to too many concurrent jobs and large datasets. The batch job side should time out the request at some point and users should probably expect the job to fail and be retried by their job scheduler, rather than the job running indefinitely waiting to be served.

Reply:
details on this will be added to readme on gRPC